app/controllers/admins/ws_panel_controller.rb

```diff
@@ -191,7 +191,7 @@ def get_driver_data
       if @driver.avatar?
         driver_data[:avatar] = @driver.avatar.url(:medium)
       else
-        driver_data[:avatar] = '/style/no_avatar.png'
+        driver_data[:avatar] = '/avatars/original/missing.png'
```

Comment:
Maybe, it's better to place `missing.png` in `thumb` and `medium` directories and forget about `else` branch at all?